src/gwt/src/org/rstudio/studio/RStudio.gwt.xml

```diff
@@ -26,8 +26,9 @@
       if (window.desktop)
          return "true";
 
-      // also true when run under QtWebEngine, which defers hook injection
-      return window.navigator.userAgent.indexOf("QtWebEngine") > 0 ? "true" : "false";
+      // also true when run under Electron or QtWebEngine, which defer hook injection
+      return ((window.navigator.userAgent.indexOf("Electron") > 0) ||
+              (window.navigator.userAgent.indexOf("QtWebEngine") > 0)) ? "true" : "false";
```

Comment:
I believe we should change this to >= 0 just to be consistent with the return values of indexOf, 0 implies that the string starts with that value.

Reply:
(if this is actually Javascript)

Reply:
Per our discussion we'll just go with the existing pattern here.